user/languages/python.md

```diff
@@ -49,17 +49,16 @@ levels.
 ```yaml
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.5-dev" # 3.5 development branch
-  - "3.6"
-  - "3.6-dev" # 3.6 development branch
-  - "3.7-dev" # 3.7 development branch
-  - "nightly"
+  - 2.6
```

Comment:
Keep quotes, here and elsewhere. `"2.6"` is a string, and `2.6` a float. They behave similarly in our applications, but they are different data structures.